api-reference/10 UI Components/dxGantt/3 Methods/exportToPdf(options).md

```diff
@@ -3,13 +3,81 @@ id: dxGantt.exportToPdf(options)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports the Gantt chart data to the PDF document.
 
 ##### return: Promise<any>
-<!-- Description goes here -->
+A Promise that is resolved after the Gantt chart is exported. It is a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">native Promise</a> or a <a href="http://api.jquery.com/Types/#Promise" target="_blank">jQuery.Promise</a> when you use jQuery.
 
 ##### param(options): Object
-<!-- Description goes here -->
+PDF export options.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The **export** method allows you to save information about the Gantt chart's layout, appearance, and tasks. This method supports the following options:
+
+- **format** {string | width?: number, height?: number } - Specifies the document's size.
+- **landscape** {boolean} - Specifies whether to use horizontal orientation for the document.
+- **fileName** {string} - Specifies the file name.
+- **exportMode** {"all", "treeList", "chart"} -  Specifies which part of the component to export (chart area, tree list area or the entire component).
+- **dateRange**: {"all" | "visible" | startDate? : Date, endDate? : Date, startIndex? : number, endIndex? : number } - Specifies the date range for which to export tasks.
```

Comment:
Does this parameter limit exported tasks down to a specific date range?

Reply:
yes